src/main/java/fi/otavanopisto/kuntaapi/server/integrations/ptv/translation/PtvOutPtvInTranslator.java

```diff
@@ -0,0 +1,278 @@
+    }
+    
+    List<VmOpenApiAreaIn> result = new ArrayList<>(areas.size());
+    
+    for (VmOpenApiArea area : areas) {
+      VmOpenApiAreaIn areaIn = translateArea(area);
+      if (areaIn != null) {
+        result.add(areaIn);
+      }
+    }
+    
+    return result;
+  }
+
+  private VmOpenApiAreaIn translateArea(VmOpenApiArea area) {
+    if (area == null) {
+      return null;
+    }
+    
+    VmOpenApiAreaIn result = new VmOpenApiAreaIn();
+    result.setAreaCodes(Arrays.asList(area.getCode()));
+    result.setType(area.getType());
+    
+    return result;
+  }
+
+  private List<V4VmOpenApiPhone> translatePhoneNumbers(List<V4VmOpenApiPhoneWithType> phoneNumbers) {
+    if (phoneNumbers == null || phoneNumbers.isEmpty()) {
+      return Collections.emptyList();
+    }
+    
+    List<V4VmOpenApiPhone> result = new ArrayList<>(phoneNumbers.size());
+    
+    for (V4VmOpenApiPhoneWithType phoneNumber : phoneNumbers) {
+      if (phoneNumber != null && "Phone".equals(phoneNumber.getType())) {
+        result.add(translatePhone(phoneNumber));
+      }
+    }
+    
+    return result;
+  }
+
+  private List<V4VmOpenApiPhoneSimple> translateFaxNumbers(List<V4VmOpenApiPhoneWithType> phoneNumbers) {
+    if (phoneNumbers == null || phoneNumbers.isEmpty()) {
+      return Collections.emptyList();
+    }
+    
+    List<V4VmOpenApiPhoneSimple> result = new ArrayList<>(phoneNumbers.size());
+    
+    for (V4VmOpenApiPhoneWithType phoneNumber : phoneNumbers) {
+      if (phoneNumber != null && "Fax".equals(phoneNumber.getType())) {
+        result.add(translatePhoneSimple(phoneNumber));
+      }
+    }
+    
+    return result;
+  }
+
+  private V4VmOpenApiPhone translatePhone(V4VmOpenApiPhoneWithType phoneNumber) {
+    if (phoneNumber == null) {
+      return null;
+    }
+    
+    V4VmOpenApiPhone result = new V4VmOpenApiPhone();
+    result.setAdditionalInformation(phoneNumber.getAdditionalInformation());
+    result.setChargeDescription(phoneNumber.getChargeDescription());
+    result.setIsFinnishServiceNumber(phoneNumber.getIsFinnishServiceNumber());
+    result.setLanguage(phoneNumber.getLanguage());
+    result.setNumber(phoneNumber.getNumber());
+    result.setPrefixNumber(phoneNumber.getPrefixNumber());
+    result.setServiceChargeType(phoneNumber.getServiceChargeType());
+    
+    return result;
+  }
+
+  private V4VmOpenApiPhoneSimple translatePhoneSimple(V4VmOpenApiPhoneWithType phoneNumber) {
+    if (phoneNumber == null) {
+      return null;
+    }
+    
+    V4VmOpenApiPhoneSimple result = new V4VmOpenApiPhoneSimple();
+    result.setIsFinnishServiceNumber(phoneNumber.getIsFinnishServiceNumber());
+    result.setLanguage(phoneNumber.getLanguage());
+    result.setNumber(phoneNumber.getNumber());
+    result.setPrefixNumber(phoneNumber.getPrefixNumber());
+
+    return result;
+  }
+
+  private List<VmOpenApiLanguageItem> translateLocalizedListItemsToLanguageItems(List<VmOpenApiLocalizedListItem> listItems) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "translateLocalizedListItemsToLanguageItems" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUnusedPrivateMethod)